test/image-blueprints/layer4-art/group1/rhel93_microshift-art.toml

```diff
@@ -0,0 +1,39 @@
+name = "rhel-93-microshift-art-base"
+description = "A RHEL 9.3 image with the RPMs built from ART."
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-93"
+
+[[packages]]
+name = "microshift"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-networking"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-selinux"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "systemd-resolved"
+version = "*"
+
```

Comment:
Can we add here the same extra packages [I commented in the rhel92 toml file](https://github.com/openshift/microshift/pull/3012/files#r1489492489)?